client/components/editor/ActivityContent/ContainerList.vue

```diff
@@ -91,26 +120,32 @@ export default {
       saveElement: 'save',
       updateElement: 'update',
       deleteElement: 'remove',
-      insertElement: 'insert',
       reorderElements: 'reorder'
     }),
     addContainer() {
       const { type, parentId, nextPosition: position } = this;
       this.save({ type, parentId, position });
     },
-    saveContentElement(element) {
-      const elementChannel = this.$radio.channel(`element:${getElementId(element)}`);
-      return this.saveElement(element)
-        .then(() => {
-          elementChannel.emit('saved');
-          this.showNotification();
-        })
-        .catch(err => elementChannel.emit('error', err));
+    saveContentElements(elements) {
+      // TODO: implement endpoint to save multiple elements at once
+      const contentElements = castArray(elements);
+      Promise.map(contentElements, element => {
+        const elementChannel = this.$radio.channel(`element:${getElementId(element)}`);
+        return this.saveElement(element)
+          .then(() => elementChannel.emit('saved'))
+          .catch(err => {
+            elementChannel.emit('error', err);
+            return Promise.reject(err);
+          });
```

Comment:
Can we flatten this promise chain by moving getting channel and emitting event through channel to `saveElement` method?

Reply:
> Here's a couple of small suggestions (mostly format-related), please see if any of these make any sense. 😄

Thank you 😉